src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+
+  if (days >= 3) {
+    if (days >= 7) {
+      return (cost - 50);
+    }
```

Comment:
that's a bad idea to nest conditions, you could get it in a simpler way, find out how
also, consider moving `20` and `50` to vars so that reviewers understand the context of what's written